libraries/Microsoft.Bot.StreamingExtensions/HttpContentStream.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net.Http;
+
+namespace Microsoft.Bot.StreamingExtensions
+{
+    /// <summary>
+    /// An attachment contained within a <see cref="StreamingRequest"/>'s stream collection,
+    /// which itself contains any form of media item.
+    /// </summary>
+    public class HttpContentStream
+    {
```

Comment:
There is a class called "ContentStream". What's the relatship with this? I would totally expect this to be:
public class HttpContentStream : ContentStream

... or at least some some similar relationship.